server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -383,9 +378,9 @@ private Pair<DivTag, ImmutableList<Modal>> renderActiveOrDraftRow(
   private DivTag renderInfoCell(QuestionDefinition definition) {
     DivTag questionText =
         div()
-            .withClasses("font-bold", "text-black", "flex", "flex-row", "items-center")
+            .withClasses("question-bank-info-cell")
             .with(
-                Icons.questionTypeSvg(definition.getQuestionType())
+                Icons.questionTypeSvgWithId(definition.getQuestionType())
```

Comment:
```suggestion
                Icons.questionTypeSvg(definition.getQuestionType())
```

Adding this id here may result in a conflict on this page. Let's hold off on doing the `/admin/questions` list for a different PR.

Reply:
After this I think we're good!

Reply:
I updated. Thanks Gwen!

Reply:
One question- But our goal was to improve the performance in the question list page, if we don't reuse the svg, won't it be against the effort?